powerhome/playbook
# [PLAY-230] Typescript Conversion: Dialog

#### Screens

<img width="1719" alt="Screen Shot 2022-09-20 at 1 17 44 PM" src="https://user-images.githubusercontent.com/8194056/191311275-b9c3b95e-feb1-4536-a338-f546b875713f.png">

#### Breaking Changes

No breaking changes, typescript conversion. Two variables were removed because they were not being used and the `Modal` component that comes from the `react-modal` library was complaining that these props didn't exist:
`alertStyle={alertStyle}` and ` status={status}`

#### Runway Ticket URL

[[PLAY-230]](https://nitro.powerhrg.com/runway/backlog_items/PLAY-230)

#### How to test this

- Run locally and try implementing the kit with different props using typscript
- Check for console errors

#### Checklist:
